### DISCUSSION TITLE
TypeError: Cannot read properties of undefined (reading 'config') directly after initialising the Next.js project

### DISCUSSION THREAD

**Discussion by 0xrishavsharma** [vercel/next.js#41347] [Help]
I am getting the following error just after initialising the project using `yarn create next-app .`
Did nothing but initialised this project. Never encountered this error except this time

``` javascript
rishavsharma@Rishavs-Air rishavsharma-blog % yarn dev
yarn run v1.22.19
warning ../../package.json: No license field
$ next dev
ready - started server on 0.0.0.0:3000, url: http://localhost:3000
error - ./node_modules/next/dist/build/webpack/loaders/css-loader/src/index.js??ruleSet[1].rules[2].oneOf[8].use[1]!./node_modules/next/dist/build/webpack/loaders/postcss-loader/src/index.js??ruleSet[1].rules[2].oneOf[8].use[2]!./styles/globals.css
TypeError: Cannot read properties of undefined (reading 'config')
```
Operating System: MacOS Monterery 12.6
Node: 16.17.1
Yarn: 1.22.19
Code Editor: VS code
Browser: Brave Browser

**Comment by 0xrishavsharma** [comment]
I had a look at the console logs and found out that this is a problem related to tailwind. But I didn't even install anything related to tailwind than why was it showing this error?

<img width="1440" alt="Screenshot 2022-10-12 at 11 23 15 AM" src="https://user-images.githubusercontent.com/63146727/195261199-f43f3e04-0b75-4d57-aa1a-c9afa6cfff71.png">

Anyway, I stumbled upon few answers in GitHub discussions and on StackOverFlow. Then I just added a `tailwind.config.js` file in the root directory and added the following lines of code:

``` javascript
module.exports = {
    content: [],
    theme: {
        extend: {},
    },
    plugins: [],
}

```

I restarted the server again and BOOM!!
The problem was solved. No more `TypeError`

BUT, this is not a solution.

**Reply by icyJoseph** [reply]
This shouldn't be the solution though. You don't have tailwind in the result of `create next-app` so this makes very little sense. How have you structured this project? I see this warning: `warning ../../package.json: No license field`, why is it looking two levels up?  

Did you create the project in one level and then moved it down? did you re-install node_modules? Very weird issue. I can't even reproduce it 😅 

I guess it is possible that you have nested projects, somehow, and running yarn dev from an intermediate level ends up running the package.json of the parent project?

**Reply by 0xrishavsharma** [reply]
You are a saviour @icyJoseph!
You are absolutely right. I created a folder in the desktop and installed everything from scratch but somehow the `package.json` was being loaded from the parent diretory(here `../../`). I deleted that file and other files that I thing got created in the past due to some mistake.

Now, the problem is solved.
I restarted the server and everything went as expected.

**Still there is a question.**
Why my project was considering the `package.json` from 2 directories above and not the one created in the root?

**Comment by icyJoseph** [comment]
This `warning ../../package.json: No license field` is very suspicious, why would the command run up above two directory levels?
